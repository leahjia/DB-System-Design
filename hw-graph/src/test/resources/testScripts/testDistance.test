# University Map Distance testing file.
# Tests the distance from one building to another in the university map

# tests the distance from building A to B in UnivMap
FidnDist Node A Node B

# tests the distance from building B to A in UnivMap
FidnDist Node B Node A
