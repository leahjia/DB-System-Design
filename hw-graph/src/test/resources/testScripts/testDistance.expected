# University Map Distance testing file.
# Tests the distance from one building to another in the university map

# tests the distance from building A to B in UnivMap
tests distance from building A to B

# tests the distance from building B to A in UnivMap
tests distance from building B to A
